lib/xmldsig/signature.rb

```diff
@@ -11,7 +11,7 @@ def initialize(signature, id_attr = nil, referenced_documents = {}, base64_stric
 
     def references
       @references ||= signature.xpath("descendant::ds:Reference", NAMESPACES).map do |node|
-        Reference.new(node, @id_attr, @referenced_documents)
+        Reference.new(node, @id_attr, @referenced_documents, @base64_strict_encode)
```

Comment:
@rwdaigle I missed this in the previous commit. The digests generated in testing have not been long enough to encounter line breaks but adding this for completeness. I also beefed up tests.